packages/core/src/awsService/appBuilder/serverlessLand/main.ts

```diff
@@ -61,9 +56,12 @@ export async function createNewServerlessLandProject(extContext: ExtContext): Pr
             },
             true
         )
+        telemetry.record({
+            runtimeString: config.runtime,
+            templateName: assetName,
+            result: 'Succeeded',
```

Comment:
As mentioned in another comment, dont need to specify result when in a `run()` context